environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -514,6 +578,9 @@ objects:
     - name: http
       port: 10902
       targetPort: http
+    - name: https
+      port: 8080
```

Comment:
hmm this is funny, are we always using 8080 for https in this repo? it doesn't really matter of course since it's all isolated in the container and used for mapping but by convention we normally use 8080 for http and something like 8443 for https

Reply:
We use `9091` for the rest of it, but I believe it's also weird. I'll use `8443`.

Reply:
ack makes sense, we use 9090 by convention for http in the prometheus-world, so 9091 is often https or any proxy. 8443 sounds :100: